### ISSUE TITLE
Error message: setContents in combination with Vue.js

### ISSUE THREAD

**Issue by MRB60** [JiHong88/SunEditor#153]
Great project, thanks!
Minor issue - receives error message but still sets contents.
After SUNEDITOR.create(..), I set the contents with setContents(..) and receives the following error:
   _TypeError: Cannot read property 'push' of null
       at Object.setContents (suneditor.min.js:1)
       at Object.setContents (suneditor.min.js:1)
       at Vue.openEditModal (pageedit.js:176)
       ...
At the moment I do a try...catch and contents is still set...but any chance this could be fixed so I don't get an error? Perhaps something to do with initializing history?

...and here is the code:
```
var HTMLeditorcontent = document.querySelector("#editarea" + id).innerHTML;
this.editorconfig.callBackSave = this.dataSave; //cannot be set in data sector
this.suneditor = SUNEDITOR.create((document.getElementById('suneditor') || 'suneditor'));
// Load the contents for the suneditor before options are set, then save starts as disabled.
// for some reason it throws an error but still does what it should do...
try {
   this.suneditor.setContents(HTMLeditorcontent);
} 
catch(err) {
   console.log('Error setting suneditor contents - still seem to do it. ERR:', err);
```

**Comment by JiHong88** [comment]
HI, @MRB60 

A bug occurred while adding the "iframe" option.

When using "iframe" in the core's initialization method, the HTML is not rendered immediately, so the editor's auto height is calculated in the "setTimeout" function.
The code that initializes the editor's history object is also in the "setTimeout" function, which causes an error when using the "setContents" method immediately after creating the editor.

The revised version will be updated this Sunday.

Thank you.

**Comment by MRB60** [comment]
Great!
Running Suneditor with CDN so hope it OK that issue closes when new dist is available there.
If not, I'll try to download and test the already updated git version...

**Comment by MarcoRampino** [comment]
Good afternoon,
i have the same problem, when it will be released?

**Comment by JiHong88** [comment]
Hi @MRB60 , @MarcoRampino 
Fixed in [v2.22.0](https://github.com/JiHong88/SunEditor/releases/tag/2.22.0)
Thank you.
